docs/client-options.md

```diff
@@ -44,6 +44,7 @@ third party clients.
 
 ## Go
 
+- [autocert] (https://godoc.org/golang.org/x/crypto/acme/autocert)
```

Comment:
Could you move this addition to after the `GoACME` entry? We've been adding new clients sequentially as opposed to alphabetically.